bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -536,7 +594,7 @@
                         }
                     }
 
-                    return cachedMethodNames;
+                    return Object.keys(objCObjectBuiltins).concat(cachedMethodNames);
```

Comment:
It isn't on the prototype, maybe we can only add them to `enumerate`, but not `keys`?